packages/MarkdownEditor-Core.package/MarkdownBlockTextStyler.class/instance/buildAttributeRunsFrom..st

```diff
@@ -0,0 +1,6 @@
+parsing
+buildAttributeRunsFrom: aCollectionOfMarkdownBlocks
+	
+	^ (aCollectionOfMarkdownBlocks 
```

Comment:
I'd prefer to put the larger expression into a variable to avoid overlooking `runs`.

```smalltalk
| text |
text := (aCollectionOfMarkdownBlocks
	inject: Text new
	into: [:text :block | text append: block asText]).
^ text runs